producer-js/browser-based-ingestion/dist/main.js.LICENSE.txt

```diff
@@ -0,0 +1,305 @@
+/*!
```

Comment:
Thanks for your contribution. I think you have accidentally commented out / renamed main.js. Please fix it